.github/workflows/bump-version.yml

```diff
@@ -19,6 +19,7 @@ jobs:
       - name: Checkout Cloudbeat Repo
         uses: actions/checkout@v4
         with:
+          ref: ${{ github.base_ref || github.ref_name }}
```

Comment:
@kubasobon I'm not entirely sure about the use of `github.base_ref` in this workflow. Since the only trigger for this workflow is `workflow_dispatch`, `github.base_ref` will not be relevant and will always be empty. We only need to use `github.ref_name` to refer to the branch being processed. As mentioned [there](https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/accessing-contextual-information-about-workflow-runs), the `github.base_ref` property is only available when the event triggering the workflow is either `pull_request` or `pull_request_target`. It will not be populated for other events like `workflow_dispatch`.